### DISCUSSION TITLE
About server performance

### DISCUSSION THREAD

**Discussion by pausto** [open-meteo/open-meteo#691] [Q&A] [answered]
Hello Patrick.
Can you tell me about the main server? There are a lot of models in the program, all of them are downloaded and processed, very often at the same time, it takes a lot of resources. Tell me what kind of server is used (CPU, disks, memory), what is the network speed and how much data is downloaded per month (or day).
Thank you

**Comment by patrick-zippenfenig** [comment, marked as answer]
Open-Meteo uses many different servers to download and process data. Servers are located close to the open-data servers from various national weather service (e.g. USA, Canada, Germany, Tokyo). This drastically improves download and conversion performance. Preferably dedicated servers with 64 or 128 GB memory are used. Of course Open-Meteo downloads and processes not only surface weather variables but also pressure level data. For many deployments, only a hand full of weather variables are relevant.
